diff --git a/node_modules/@react-native-community/clipboard/RNCClipboard.podspec b/node_modules/@react-native-community/clipboard/RNCClipboard.podspec
index cde3931..ed0b5bb 100644
--- a/node_modules/@react-native-community/clipboard/RNCClipboard.podspec
+++ b/node_modules/@react-native-community/clipboard/RNCClipboard.podspec
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/react-native-community/clipboard", :tag => "#{s.version}" }
   s.source_files  = "ios/**/*.{h,m,mm}"
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
