diff --git a/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec b/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec
index a1bd5cb..df00a60 100644
--- a/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec
+++ b/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/react-native-community/react-native-async-storage.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
