diff --git a/node_modules/react-native-keychain/RNKeychain.podspec b/node_modules/react-native-keychain/RNKeychain.podspec
index 908a5c6..dd5b33f 100644
--- a/node_modules/react-native-keychain/RNKeychain.podspec
+++ b/node_modules/react-native-keychain/RNKeychain.podspec
@@ -14,6 +14,6 @@ Pod::Spec.new do |s|
   s.source         = { :git => "https://github.com/oblador/react-native-keychain.git", :tag => "v#{s.version}" }
   s.source_files   = 'RNKeychainManager/**/*.{h,m}'
   s.preserve_paths = "**/*.js"
-  s.dependency 'React'
+  s.dependency 'React-Core'
 
 end
