diff --git a/node_modules/react-native-document-picker/index.js b/node_modules/react-native-document-picker/index.js
index 7e49bad..fe13205 100644
--- a/node_modules/react-native-document-picker/index.js
+++ b/node_modules/react-native-document-picker/index.js
@@ -81,7 +81,7 @@ const Types = {
     zip: 'application/zip',
   },
   utis: {
-    allFiles: 'public.content',
+    allFiles: 'public.data',
     audio: 'public.audio',
     csv: 'public.comma-separated-values-text',
     images: 'public.image',
