diff --git a/node_modules/react-native-touch-id/TouchID.podspec b/node_modules/react-native-touch-id/TouchID.podspec
index cb59f8b..426aa88 100644
--- a/node_modules/react-native-touch-id/TouchID.podspec
+++ b/node_modules/react-native-touch-id/TouchID.podspec
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/naoufal/react-native-touch-id.git" }
 
   s.source_files  = "*.{h,m}"
-  s.dependency "React"
+  s.dependency "React-Core"
 end
